monsteer/qt/SteeringWidget.h

```diff
@@ -40,15 +40,10 @@ class SteeringWidget : public QWidget
 public:
 
     /**
-     * @param selectionZeqSchema the ZeroEQ schema used for receiving selection
-     *        events.
-     * @param simulationZeqSchema the ZeroEQ schema used for sending stimuli to
-     *        the simulation.
+     * Constructor
      * @param parentWgt Parent widget.
      */
-    SteeringWidget( const servus::URI& selectionZeqSchema,
-                    const servus::URI& simulationZeqSchema,
-                    QWidget *parentWgt = 0 );
+    SteeringWidget( QWidget *parentWgt = 0 );
```

Comment:
I am not sure if that's going to be useful for a GUI widget, we will add the functionality through the UI, and one can also use the ENV variable as an alternative.  We can add it later if we see that we need it.